tests/unit/types/arrays/test_neural_ops.py

```diff
@@ -433,8 +433,14 @@ def test_embeddings_setter():
     da.embeddings = emb
     np.testing.assert_almost_equal(da.embeddings, emb)
 
+    for x, y in zip(emb, da.embeddings):
```

Comment:
THis is not what I mean.
I mean for `for emb, doc in zip(embeddings, da)` assert `doc.embedding == emb` or something like this

Reply:
Ok, updated with the doc.embedding test